app/Http/Controllers/GroupController.php

```diff
@@ -0,0 +1,165 @@
+        });
+
+        return view('groups.admin', compact('groups', 'courses'));
+    }
+
+    public function create()
+    {
+        $student = Auth::student();
+
+        $courses = $student->getCoursesWithoutGroup();
+
+        return view('groups.create', compact('courses'));
+    }
+
+    /**
+     * 
+     */
+    public function store(Request $request) {
+        
+        $atributtes = $request->validate([
+            'email' => 'required',
+            'course_id' => 'required'
+        ]);
+
+        DB::transaction(function () use ($atributtes){
+            $creator = Auth::student();
+            $course = Course::findOrFail($atributtes['course_id']);
+
+            $group = new Group();
+            $course->addGroup($group);
+            $group->refresh();
+
+            $group->addMember($creator);
+            $creator->confirmGroup($group);
+
+            $user = User::where('email', $atributtes['email'])->first();
+            $studentToAdd = $user->student()->first();
+            $group->addMember($studentToAdd);
+
+            return $group;
+
+        });
+
+        flash()->success('New group created successfully!');
+
+        return redirect()->route('groups.index');
+
+    }
+
+    public function edit(Group $group) {
+        return view('groups.edit', compact('group'));
+    }
+
+    public function sendInvite(Group $group, Request $request) {
+        $atributtes = $request->validate([
+            'email' => 'required',
+        ]);
+        $user = User::where('email', $atributtes['email'])->first();
+        $studentToAdd = $user->student()->first();
+        $group->addMember($studentToAdd);
+
+        flash()->success('Invite sent successfully!');
+        return redirect()->route('groups.index');
+    }
+
+    public function update(Group $group) {
+        $student = Auth::student();
+        $student->confirmGroup($group);
+        flash()->success('Invite accepted successfully!');
+        return redirect()->route('groups.index');
+    }
+
+    public function destroy(Group $group) {
+        $student = Auth::student();
+        $student->declineGroup($group);
+        return redirect()->route('groups.index');
+    }
+
+    public function leave(Group $group) {
+        $student = Auth::student();
+        $student->leaveGroup($group);
+        return redirect()->route('groups.index');
+    }
+
+    /**
+     * Exports the list of groups in each course.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function export()
+    {
+
+        $groups = DB::transaction(function () {
+            $groups = Group::with('course','students')
+                ->get()
+                ->sortByDesc('courses.name');
+
+            return $groups;
+        });
+
+        $filename = "movies.json";
+        $handle = fopen($filename, 'w+');
+        fputs($handle, $groups->toJson(JSON_PRETTY_PRINT));
+        fclose($handle);
+        $headers = array('Content-type'=> 'application/json');
```

Comment:
Remove this line and to new array syntax `['Content-type' => 'application/json']`.

Reply:
And I'm not sure if you can use this syntax at the response `$response()->header('Content-Type', 'application/json'];`
https://laravel.com/docs/5.5/responses#view-responses